src/Bicep.Core/Emit/ExpressionConverter.cs

```diff
@@ -664,8 +664,10 @@ private static LanguageExpression ConvertFunction(string functionName, IEnumerab
         {
             if (string.Equals("any", functionName, LanguageConstants.IdentifierComparison))
             {
+                //miqm: I think this is not used, we're catching any function in ExpressionEmitter.cs:149
```

Comment:
>ExpressionEmitter [](http://example.com/codeflow?start=81&length=17)

We can confirm this with code coverage. If this path/block doesn't show up as covered with all our tests, we can probably remove it.

Reply:
oddly, it's being shown as covered. This means that my evaluator might not work in all cases. Need to dig deeper...

Reply:
It's being called from within the ExpressionConverter itself.

It seems to me like ExpressionEmitter and Converter are both doing similar job and sometimes function from Emitter is used, and sometimes from Converter. What's the rule here?

Reply:
I moved the logic for Emitting Function from Emitter to Converter. It seems that code for handling `any` in the Emitter was a short-circuit to avoid calling expressionConverter. It works now, coverage didn't show that it's being used. Resolving.